cmake/Modules/FindGflags.cmake

```diff
@@ -0,0 +1,66 @@
+###############################################################################
+#
+# CMake script for finding the GFlags library.
+#
+# https://gflags.github.io/gflags/
+#
+# Redistribution and use is allowed according to the terms of the 3-clause BSD
+# license.
+#
+# Input variables:
+#
+#
+# Cache variables (not intended to be used in CMakeLists.txt files)
+#
+# - _gflags_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES
+# - CMAKE_FIND_LIBRARY_SUFFIXES: ?
+#
+# Output variables:
+#
+# - Gflags_FOUND: Boolean that indicates if the package was found
+# - Gflags_LIBRARIES: shared objects exported from gflags
+# - Gflags_INCLUDE_DIRS: this is exported from somewhere else magically
```

Comment:
Also though documented here, this variable is never set below, which I guess is why you might consider it magical, but I see no way that it gets set by this file. So I'd appreciate it if you could explain how `Gflags_INCLUDE_DIRS` gets set.